packages/common/src/types.ts

```diff
@@ -8,11 +8,11 @@ export interface genesisStatesType {
   [key: string]: {}
 }
 
+export interface nameType {
```

Comment:
`nameType` isn't very descriptive, maybe `ChainName`? would you be able to define it as `[chainId: string]: string`